client/themes/morning.css

```diff
@@ -46,8 +46,22 @@ body {
 	padding: 15px 16px;
 }
 
+/* stylelint-disable */
```

Comment:
I'm confused, why do we have to add all these exceptions actually? It's kind of a shame :(